tests/server/master_server/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("@rules_cc//cc:defs.bzl", "cc_test")
+
+cc_test(
+    name = "chunk_server_manager",
+    srcs = ["chunk_server_manager.cc"],
+    copts = ["-Iexternal/gtest/include", "-Isrc/server/master_server"],
```

Comment:
Let's not use the absolute include for `server/master_server`. Instead, in `server/master_server_manager.cc`, make the header an absolute path